docs/org.md

```diff
@@ -1,31 +1,30 @@
 ## Introduction
 
-Lychee was originally developped by [electerious][1] ([Tobias Reich][2]). Lychee aims to be a great looking and easy-to-use photo-management-system that nearly anyone can use on their webserver.
+Lychee was originally developed by [electerious][1] ([Tobias Reich][2]). Lychee aims to be a great looking and easy-to-use photo-management-system that nearly anyone can use on their web server.
 
 Since April 1st, 2018 this project has moved to it's own Organisation (LycheeOrg) where people are able to submit their fixes to it. We, the Organisation owners, want to thank electerious (Tobias Reich) for the opportunity to make this project live on.
 
 ## Schedule releases & Roadmap.
 
-We do not have schedule for releases such as every 6 months, however we still follow the [semantic versioning][3]. Our releases should never contain breaking changes. That being said, we do not provide backward compatibility or hotfix for older versions. The `master` branch is the bleeding edge of Lychee. It contains the latest bugfix and newest features. Once a sufficient number of hotfixes or new features and has been reached we release a minor version.
+We do not have schedule for releases such as every 6 months. Our releases should never contain breaking changes. That being said, we do not provide backward compatibility or hotfix for older versions. The `master` branch is the bleeding edge of Lychee. It contains the latest bugfix and newest features. Once a sufficient number of hotfixes or new features has been merged into `master` we create a new release.
 
 ## Security
 
 In order to ensure maximum security some actions on Lychee are restricted by an admin authentication. The username and password are stored hashed with bcrypt in the database.
 
-However as precaution, we provide a serverside command to easily erase those two data from the database in order to recover admin access. We consider this command safe as it requires a command line access to the server. A user gaining such rights is outside of our security scope as the server would be compromised.
+However as precaution, we provide a server-side command to easily erase those two data from the database in order to recover admin access. We consider this command safe as it requires a command line access to the server. A user gaining such rights is outside of our security scope as the server would be compromised.
 
 ## LycheeOrg
 
-LycheeOrg is a github organization grouping enthousiast developers determined to keep this project alive.
-There is no commitment in time to the project. We understand that each member has their personnal life and dedicate time as they see fit to the project.
+LycheeOrg is a GitHub organization grouping enthusiastic developers determined to keep this project alive.
```

Comment:
Actually, I believe this should say _enthusiast_.

Reply:
I considered that, too, because it has a smaller editing distance (one character only). But, IMHO, this must be a adjective. "Developers" is already a noun and two nouns following each other does not work grammatically.

It is neither an enumeration, because then we would need a conjunction like "and" in "enthusiasts _and_ developers".

Reply:
IMO either could work but mean different things. I'd lean towards "enthusiast" though.

If we're going down this line I'd change "grouping" to "of" as well, but I don't feel especially strongly about it.

Reply:
You are a native speaker, right? What are the two different meanings you see here? For my German brain the combination of the two nouns just doesn't make any sense.

And yes, if you have spotted more errors, feel free to correct them. I just corrected what caught my eye.

Reply:
> What are the two different meanings you see here?

"enthusiast developer" is like "hobbyist developer" or "amateur developer". Mostly, "we're not being paid for this.", which I feel is the main point.
"enthusiastic developer" is one full of enthusiasm - "excited developer" maybe.

Reply:
Ah, OK, so in the first version "enthusiast" is still used as an adjective.

For me it was like "car manufacturer", someone who manufactures cars, "train conductor", someone who conducts a train. Likewise, a "enthusiast developer" would be someone who develops enthusiasts, which doesn't make much sense. That is at least what a German brain would make out of it.

Anyway, I deem this discussion resolved. I will change it as you propose to "enthusiast developer".